students/nartowska_karina/lesson_12_debugging/validated_user_base.py

```diff
@@ -0,0 +1,66 @@
+import os
+import argparse
+import csv
+import logging
+
+from validators import email_validator
+from validators import postal_code_validator
+from validators import phone_number_validator
+from validators import strong_password_detection
+
+LOGGER_LEVELS = {"debug": logging.DEBUG,
+                 "info": logging.INFO,
+                 "warning": logging.WARNING,
+                 "error": logging.ERROR}
+
+
+def check_arg(args=None):
+    parser = argparse.ArgumentParser(description='Validated user base')
+    parser.add_argument('-v', '--verbose',
+                        help='verbose output: debug, info, warning, error',
+                        default='debug')
+
+    result = parser.parse_args(args)
+    return result.verbose
+
+
+verbose = check_arg()
+logging.basicConfig(level=LOGGER_LEVELS[verbose],
+                    format='%(asctime)s - %(levelname)s - %(message)s')
+
+
+def combined_validator(email, password, phone_number, postal_code):
+    if not (email_validator(email) and
+            strong_password_detection(password) and
+            phone_number_validator(phone_number) and
+            postal_code_validator(postal_code)):
+        raise Exception("Incorrect data")
+
+
+def write_to_csv(input):
+    with open('data.csv', 'a+') as csv_file:
+        fieldnames = ['email', 'password', 'phone_number', 'postal_code']
+        writer = csv.DictWriter(csv_file, fieldnames=fieldnames)
+
+        if not os.stat(csv_file.name).st_size:
+            writer.writeheader()
+        writer.writerow(input)
```

Comment:
According to the task description, there shouldn't be possibility to insert data with the same address more than one time:

If a record with the same email is already in the file, the old record should be altered by new one